kinoml/core/proteins.py

```diff
@@ -109,9 +109,17 @@ class ProteinStructure(BaseProtein, BaseStructure):
     def from_file(cls, path, ext=None, **kwargs):
         from MDAnalysis import Universe
         from pathlib import Path
+
+        identifier = f"{path.split('/')[-1].split('.')[0]}"  #  set id to be the file name
```

Comment:
If you are using `pathlib.Path`, you can do this instead! 

```suggestion
        path = Path(path)
        identifier = path.stem
```

Reply:
`path.stem` only removes last extension though.  If you want to cover that case, use:

```
identifier = path.name.split('.')[0]
```

Reply:
This is great!